modules/alloy/src/main/ts/ephox/alloy/positioning/mode/ContentAnchorCommon.ts

```diff
@@ -9,19 +9,9 @@ import * as Origins from '../layout/Origins';
 import { Anchoring, NodeAnchor, nu as NuAnchor, SelectionAnchor } from './Anchoring';
 import * as AnchorLayouts from './AnchorLayouts';
 
-const capRect = (left: number, top: number, width: number, height: number): Optional<Boxes.BoxByPoint> => {
```

Comment:
This logic here was only a workaround to a larger issue and only resolved it for things rendered near the top of the screen, not the bottom. It's now better handled within Bounder by its constraints/restrictions.